### PR TITLE
Message zalo ui

### PR DESCRIPTION
**Pull request by ceuli**

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -4,7 +4,15 @@ import { InMemoryCache } from 'apollo-cache-inmemory'
...
-import { APOLLO_SERVER } from './configs.local'
+import { APOLLO_SERVER, APOLLO_SOCKET } from './configs.local'
+import {
+	GET_DRAFT_LIST,
+	GET_NEW_NOTI_MESSAGE_LIST,
+} from './views/Chat/gql/query'
+
+import { split } from 'apollo-link'
+import { WebSocketLink } from 'apollo-link-ws'
+import { getMainDefinition } from 'apollo-utilities'
```
Comment (michaelnguyen2401) [inline src/client.js:16]: Should import global packages at the top of the file

File: src/client.js [hunk 2]
```diff
@@ -124,6 +220,23 @@ const initialState = {
...
+	selectedUserOfChat: {
+		id: '',
+		displayName: '',
+		avatar: '',
+		__typename: 'UserOfChat',
+	},
+
+	draftList: {
+		items: [],
+		__typename: 'draftList',
```
Comment (michaelnguyen2401) [inline src/client.js:233]: Should use Capitalized Case for __typename

File: src/client.js [hunk 3]
```diff
@@ -124,6 +220,23 @@ const initialState = {
...
+	},
+
+	newNotiMessageList: {
+		items: [],
+		__typename: 'newMessageNotiList',
```
Comment (michaelnguyen2401) [inline src/client.js:238]: Should use Capitalized Case for __typename

**Comment by leovo178** [comment]
Need to follow coding convention closely
- should not use _export default function EditorChat({ idUser })_ or _export default function GroupMessage({items, listPureMessage, type, timestamp, meId, endOfGroup,})_

**Comment by leovo178** [comment]
consolidate import style to use alias, ex
- _import { ON_ZALO_MESSAGE_RECEIVED } from '../../../../Chat/gql/subscription'_

**Pull request merged**